pkg/cmd/tui.go

```diff
@@ -187,7 +188,7 @@ func runTUI(ko *KubeOptions, do displayOptions) error {
 		// table here. This TUI library needs us to build tables from a 2D array.
 		// The CSV-rendered (string) go-pretty table, nicely sorted and everything,
 		// is parsed into a 2D array and then the TUI table is built from that.
-		tWriter := makeAggregationRateTable(aggs, aggregationOptions[aggregation].headers, aggregationOptions[aggregation].titleExtractor, do, currencyCode)
+		tWriter := makeAllocationTable("Namespace", allocations, do, currencyCode, false, false)
```

Comment:
I think we have to revisit this. It depends on some options https://github.com/kubecost/kubectl-cost/blob/cfc068e80149f0df107400d2ca7f2d133c90c0ba/pkg/cmd/tui.go#L43-L71 that we set with https://github.com/kubecost/kubectl-cost/blob/main/pkg/cmd/tui.go#L165-L166

You may be able to do
```
makeAllocationTable(aggregation, allocations, do, currencyCode, false, true)
```

(you need to set the last to false because the TUI calls it "monthly rate")

Reply:
I was assuming the wrong semantics. for the switch of the last arg from `false` to `true`, but see my later comments about that arg.